gst/nnstreamer/tensor_transform/README.md

```diff
@@ -1 +1,79 @@
 # NNStreamer::tensor_transform
+
+## Supported Features
+
+- Transformation the shape, data values (arithmetics or normalization), or data type of ```other/tensor``` stream.
+- If possible, the tensor_transform element exploits [ORC: Optimized inner Loop Runtime Compiler](https://gitlab.freedesktop.org/gstreamer/orc) to accelerate the supported operations.
+- Aggregate multiple operators into a single transform instance for performance optimization.
+  - E.g., ```tensor_transform mode=typecast option=uint8 ! tensor_transform mode=arithmetic option=mul:4 ! tensor_transform mode=arithmetic option=add:25 can be optimized by tensor_transform mode=arithmetic option=typecast:uint8,mul:8,add:25```
+
+## Planned Features
+
+- TBD
+
+## Pads
+
+- SINK
+
+   - One always sink pad named 'sink'
+   - other/tensor
+
+- SRC
+
+   - One always source pad named 'src'
+   - other/tensor
+
+## Properties
+
+- mode (readable, writable): Mode used for transforming tensor
+  - Enum "gtt_mode_type" Default: -1, "unknown"
```

Comment:
> http://nnsuite.mooo.com/nnstreamer/ci/repo-workers/pr-checker/2111-202002181711090.57120490074158-9f1a68178074c31d11683a4a6e1f46281d429148/report/misspelling_result.txt

Looks good, BTW, what is meaning of  the "**gtt**" of "gtt_mode_type"?

Reply:
@leemgs GstTensorTransform?